packages/datagrid/src/components/DatasetSerializer/datasetSerializer.test.js

```diff
@@ -166,6 +167,37 @@ describe('#getColumnDefs', () => {
 
 		expect(columnDefs).toEqual([]);
 	});
+
+	it('should returns an empty columns definitions', () => {
+		const schemaWithOptionalType = {
+			schema: {
+				type: 'record',
+				name: 'StringArrayRecord',
+				fields: [
+					{
+						name: 'field0',
+						doc: 'Nom de la gare',
+						type: [
+							'null',
+							{
+								type: 'string',
+								dqType: 'FR Commune',
+								dqTypeKey: 'FR_COMMUNE',
+							},
+						],
+						'@talend-quality@': {
+							0: 0,
+							1: 38,
+							'-1': 62,
+						},
+					},
+				],
+			},
+		};
+		const columnDefs = getColumnDefs(schemaWithOptionalType);
+
+		expect(columnDefs).toMatchSnapshot();
```

Comment:
please check the column definition is empty

Reply:
It's already test on this: https://github.com/Talend/ui/pull/1239/files/d5e90a69dcf0884fd5a9ed4ca4024621b96be90f#diff-d1f9322beedd4c9403ff8fd77387a793L164